snap/snapcraft.yaml

```diff
@@ -122,14 +112,12 @@ parts:
       echo "Starting at $(date)"
       START_TIME=$(date +%s)
 
-      CPU_COUNT=$(lscpu --parse=CPU | grep -v "^#" | wc -l)
-
-      export DESTDIR="$(readlink --canonicalize ${SNAPCRAFT_PART_INSTALL})"
+      export DESTDIR="${SNAPCRAFT_PART_INSTALL}"
```

Comment:
Please check if this makefile accepts `DESTDIR` as an environment variable since the traditional way to do that is running `make DESTDIR=/foo`.

Reply:
And here also
```make
PREFIX ?= /usr/local
```